collector/benchmarks/README.md

```diff
@@ -143,7 +143,88 @@ Rust code being written today.
 - **regex**: See above. This is an older version of the crate.
 - **piston-image**: See above. This is an older version of the `image` crate.
 - **style-servo**: An old version of Servo's `style` crate. A large crate, and
-  one used by old versions of Firefox.
+  one used by old versions of Firefox. Built with `--features=gecko`.
 - **syn**: See above. This is an older version (0.11.11) of the crate.
 - **tokio-webpush-simple**: A simple web server built with a very old version
   of tokio. Uses futures a lot, but doesn't use `async`/`await`.
+
+# How to update/add/remove benchmarks
+
+## Add a new benchmark
+
+- Decide on which category it belongs to.
+- If it's a third-party crate:
+  - If you are keen: talk with a maintainer of the crate to see if there is
+    anything we should be aware of when using this crate as a compile-time
+    benchmark.
+  - Look at [crates.io](https://crates.io) to find the latest (non-prerelease) version.
+  - Download it with `collector download -c $CATEGORY crate $NAME $VERSION`.
+    The `$CATEGORY` is probably `primary`.
```

Comment:
For those unfamiliar this might just be confusing. It's probably fine though if we add a short summary of how to decide on a category like I suggested above.